docs/running-headscale-container.md

```diff
@@ -109,7 +102,7 @@ not work with alternatives like [Podman](https://podman.io). The Docker image ca
 
     ```shell
     docker exec headscale \
-    headscale users create myfirstuser
+      headscale users create myfirstuser
```

Comment:
need to remove `headscale` here right?

```suggestion
      users create myfirstuser
```

Reply:
For `exec` the command is actually `headscale ...`, I tested the commands with docker on Debian 12.